packages/dx-react-chart-bootstrap4/src/templates/tooltip/arrow.jsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const Arrow = ({
+  arrowProps, className, placement, ...restProps
+}) => (
+  <div className={classNames('arrow', className)} {...arrowProps} {...restProps} />
```

Comment:
```jsx
export const Arrow = ({
  className, ...restProps
}) => (
  <div className={classNames('arrow', className)} {...restProps} />
```

Reply:
It is not working. The `arrowProps` should be spread.

Reply:
Should we ignore `placement`?

Reply:
We can, because of `react-popper` filter the unnecessary properties. But what if it stop filter?